robottelo/ui/locators/base.py

```diff
@@ -1754,22 +1758,23 @@
         "//button[contains(@ng-show, 'list') and "
         "contains(@ng-click, 'removeRepositories')]"),
     "contentviews.repo_search": (
-        By.XPATH, "//input[@ng-model='repositorySearch']"),
+        By.XPATH, "//input[@ng-model='table.searchTerm']"),
     "contentviews.promote_button": (
         By.XPATH,
-        ("//table[@bst-table='table']//tr/td[contains(., '%s')]"
-         "/following-sibling::td[@class='col-sm-2']/button")),
+        ("//td/a[contains(., '%s')]/following::td[@class='col-sm-2'][1]"
+         "//span[text()='Promote']")),
     "contentviews.env_to_promote": (
         By.XPATH,
         "//input[@ng-model='item.selected']/parent::label[contains(., '%s')]"),
     "contentviews.promote_version": (
         By.XPATH, "//button[@ng-click='verifySelection()']"),
     "contentview.version_filter": (
-        By.XPATH, "//input[@ng-model='filterTerm' and @placeholder='Filter']"),
+        By.XPATH, "//input[@ng-model='table.searchTerm' and "
```

Comment:
Oh, and you even changed the code to use `kt_search` instead in some places. Please, double check if it was replaced everywhere and remove this entry.